qiita_pet/uimodules/prep_template_tab.py

```diff
@@ -39,9 +39,11 @@ def _get_accessible_raw_data(user):
     raw_data
     """
     d = {}
```

Comment:
No, note that I'm not creating a list. The only reason why I'm asking if the rdid exists in the dictionary or not is to avoid all the DB accesses performed by this call:
```python
d[int(rdid)] = Study(RawData(rdid).studies[-1]).title
```
which are at least 4.